MarkdownKit/Sources/Common/Elements/Italic/MarkdownItalic.swift

```diff
@@ -45,4 +52,15 @@ open class MarkdownItalic: MarkdownCommonElement {
 
     attributedString.deleteCharacters(in: match.range(at: 2))
   }
+
+   private func attributedString(
+     _ attributedString: NSAttributedString,
+     hasAttribute attribute: NSAttributedString.Key,
+     at range: NSRange
+   ) -> Bool {
+     let attributedSubstring = attributedString.attributedSubstring(from: range)
+     var effectiveRange = NSRange(location: 0, length: 0)
+     let atributes = attributedSubstring.attributes(at: 0, effectiveRange: &effectiveRange)
+     return atributes.keys.contains(attribute)
```

Comment:
nit: (assuming we don't need to check the whole range) I think it can be simplified to:
```swift
attributedString.attribute(attribute, at: range.position, effectiveRange: nil) != nil
```